packages/ui/src/components/Visualization/ContextToolbar/ExportDocument/MarkdownComponentMapping.tsx

```diff
@@ -0,0 +1,34 @@
+import { Components } from 'react-markdown';
+import { Content, ContentVariants } from '@patternfly/react-core';
+import { Table, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+import './ExportDocument.scss';
+
+export const markdownComponentMapping: Components = {
+  p: ({ children }) => <Content component={ContentVariants.p}>{children}</Content>,
+
+  h1: ({ children }) => <Content component={ContentVariants.h1}>{children}</Content>,
+  h2: ({ children }) => <Content component={ContentVariants.h2}>{children}</Content>,
+  h3: ({ children }) => <Content component={ContentVariants.h3}>{children}</Content>,
+  h4: ({ children }) => <Content component={ContentVariants.h4}>{children}</Content>,
+  h5: ({ children }) => <Content component={ContentVariants.h5}>{children}</Content>,
+  h6: ({ children }) => <Content component={ContentVariants.h6}>{children}</Content>,
+
+  table: ({ children }) => (
+    <Table className="export-document-preview-table" borders isStriped isStickyHeader>
+      {children}
+    </Table>
+  ),
+  thead: ({ children }) => <Thead>{children}</Thead>,
+  tbody: ({ children }) => <Tbody>{children}</Tbody>,
+  tr: ({ children }) => <Tr isBorderRow>{children}</Tr>,
+  th: ({ children }) => (
```

Comment:
Maybe here you can pass the `CSS` classes you need instead of the override.

Reply:
I just tried it but the patternfly style wins... 
![Screenshot From 2025-02-14 11-41-10](https://github.com/user-attachments/assets/3fa70702-d008-468f-9378-41722dcb64d5)

Reply:
This is BTW to avoid table headers from truncating the header text

Reply:
If I add `!important` it works

Reply:
done :+1: